v1/upload.html

```diff
@@ -17,117 +17,105 @@
   <section class="section-spaces">
     <div class="container">
       <div class="row">
-        <div class="col-md-6">
-          <p class="select-text text-center select-file">Please select a file</p>
-          <br> 
-          <form action="" id="upload-form" method="POST" enctype="multipart/form-data">
-            <input type="submit" value="Upload File to S3" class="btn btn-info filter-scale" disabled>
-          </form>
-          <div class="progress hide">
-            <div class="bar"></div>
-            <div class="percent">0%</div>
+        <div class="col-md-12">
+          <div class="col-md-6 col-md-offset-3 col-sm-8 col-sm-offset-2 col-xs-10 col-xs-offset-1">
+            <p class="select-text select-file text-center">Please select a file</p>
+            <br>
           </div>
-          <h5 id="status"></h5>
-        </div>
-        <div class="col-md-6">
-          <p class="select-text text-center">Detail Description</p>
-          <div>
-            <!-- Nav tabs -->
-            <ul class="nav nav-pills multiple-tabs">
-              <li class="active">
-                <a href="#home" data-toggle="tab">About S3 </a></li>
-                <li ><a href="#profile" data-toggle="tab">Deatils</a></li>
-              </ul>
-              <br>
-              <br>
-              <br>
-              <div class="tab-content">
-                <div class="tab-pane active" id="home">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum</div>
-                <div class="tab-pane" id="profile">Sed ut perspiciatis unde omnis iste natus error sit voluptatem  qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?</div>
-              </div>
+          <div class="col-md-6 col-md-offset-4 col-sm-8 col-sm-offset-2 col-xs-10 col-xs-offset-1">
+            <form action="" id="upload-form" method="POST" enctype="multipart/form-data" class="">
+              <input type="submit" value="Upload File to S3" class="btn btn-info filter-scale" disabled>
```

Comment:
append fields to new div and this will give to provision to style them irrespective of order problem